### DISCUSSION TITLE
【2023.2.15】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#2037] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by starnop** [comment]
0.9.2 release progress synchronization

**Reply by starComingup** [reply]
k0s progress sync

**Reply by zhy76** [reply]
e2e test progress sync

**Reply by YTGhost** [reply]
kyverno integrate sync

**Comment by kakaZhou719** [comment]
2月15日 【会议纪要】

Starnop:
同步0.9.2 release 各项进展。

starComingup: k0s,k3s集成进展同步
1. k0s进入测试阶段


jsparter: sealer 使用 oci registry 进展同步
1. 目前进入code阶段，完成率约20%

cubxxw:
1. 支持非root用户部署集群,目前进展60%，流程已经走通

zhy76:同步sealer e2e 开发进展
1. e2e github 触发加入到 pr check list中，目前已经支持comment触发。
2.本地环境（mac,linux,win）支持运行UT test 和E2E test。


YTGhost: 同步kyverno开发进展，以及使用文档。
1. 基于kyveno基础镜像（非hack docker 的k8s底座）需重做。需要@王飞联系协助。
2. kyveno在运行中获取image list生成policy方案待设计。

其他事项:
1. sealer sig 划分，暂定为app sig 和cluster sig。
2. sig负责人需要本周五（2.17）前提交sig proposal
